apps/trading-e2e/src/integration/capsule.cy.ts

```diff
@@ -410,12 +410,18 @@ describe('capsule', { tags: '@slow' }, () => {
 
   it('approved amount is less than deposit', function () {
     // 1001-DEPO-006
-
     cy.getByTestId(depositsTab).click();
     cy.getByTestId('deposit-button').click();
     cy.get(assetSelectField, txTimeout).select(btcName, { force: true });
+    cy.contains('Deposits of tBTC not approved').should('not.exist');
+    cy.contains('Use maximum').should('be.visible');
     cy.get(amountField).clear().type('20000000');
-    cy.getByTestId('deposit-approve-submit').should('be.visible');
+    cy.getByTestId(depositSubmit).should('be.visible');
+    cy.getByTestId(depositSubmit).click();
+    cy.getByTestId('input-error-text').should(
+      'contain.text',
+      'Amount is above approved amount.Update approve amount'
```

Comment:
I don't understand this text.

Reply:
Is this the input error we have? maybe we should update with 
```suggestion
      'Amount is above the approved amount. Update the amount'  ? 
```